sections/appendix/locking.tex

```diff
@@ -0,0 +1,11 @@
+\subsection{Locking the gibbs sampler}\label{sec:appendix/locking}
+Normally when working with topic modelling, one does know which topics will be present in the document set before training the model.
+However, the taxonomy metadata fields provides some general subject names of different levels of abstraction and some amount of documents attached to these subject names.
+This provides a unique opportunity of using the existing taxonomies as higher-level topics.
+
+The pachinko model normally samples, like the Gibbs sampler, each taxonomy level and estimates the different topics for each document.
+
+However, in our case, we have a partially observed variable and we want to use the existing meta-information to estimate the topics quicker and more accurately.
+When we sample in the \gls{pam}, we only sample the unobserved variables and let the observed variables be. 
+This has the effect of letting the unobserved variable fit the existing topics within the model and making the process of sampling faster.
+However, locking these variables into places also hinders further improvement to the observed variables, since they do not change during training.   

```

Comment:
```suggestion
However, locking these taxonomies into places also hinders further improvement to the observed taxonomies, since they do not change during training.   

```